cloud/db/schema/environments.ts

```diff
@@ -0,0 +1,33 @@
+import { relations } from "drizzle-orm";
+import { pgTable, text, timestamp, uuid, unique } from "drizzle-orm/pg-core";
+import { projects } from "./projects";
+
+export const environments = pgTable(
```

Comment:
I'm unclear on the intentions for environments. Why do we need them and how are they distinct from projects? Why do projects need multiple environments? Could we solve the same need more powerfully via nested projects?

Reply:
SG, resolving.